zyte_common_items/util.py

```diff
@@ -65,3 +66,15 @@ def url_to_str(url: Union[str, _Url]) -> str:
             f"{url!r} is neither a string nor an instance of RequestURL or ResponseURL."
         )
     return str(url)
+
+
+def validate_timezone(value: Optional[str] = None):
+    """Validates the value against valid ZoneInfo or optional"""
+    if not value:
+        return None
+    try:
+        _ = pytz.timezone(value)
+    except pytz.UnknownTimeZoneError:
+        raise ValueError(f"{value} is not a valid timezone")
```

Comment:
Do we know for a fact that pytz.timezone only allows tzdata values?

Reply:
Did some research, only thing better than pytz is ZoneInfo but requires python3.9+. 
We could switch to ZoneInfo and use backports.zoneinfo for python3.7 / 3.8.